ports/servoshell/desktop/minibrowser.rs

```diff
@@ -282,6 +270,17 @@ impl Minibrowser {
                     let Some(servo_fbo) = servo_framebuffer_id else {
                         return;
                     };
+
+                    if let Some(status_text) = &self.status_text {
+                        egui::containers::popup::show_tooltip_at(
+                            ctx,
+                            LayerId::new(Order::Tooltip, "tooltip layer".into()),
```

Comment:
I skimmed through the egui source and I don't think we need to create the layer explicitly here. Instead we can just[ pass the parent UI's ID](https://github.com/emilk/egui/blob/8bf498d2bf849892ec0c3fa49fea4c3c504a06fe/crates/egui/src/containers/popup.rs#L43) using
```suggestion
                            ui.layer_id(),
```

egui internally creates the tooltip with the Order::Tooltip when painting the tooltip within the parent layer.